testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -74,9 +74,8 @@ Feature: Register a Salt system to be managed via SSH tunnel
     When I enter command "echo 'My remote command output'"
     And I enter the hostname of "ssh_minion" as "target"
     And I click on preview
-    Then I wait until I do not see "Target systems (1)" text
-    When I wait until I do not see "pending" text
```

Comment:
why not keep the wait for "pending" to disappear?

Reply:
It stumbled in this step, waiting until timeout. Either this text do not appear anymore or is processed this fast that is not registered. Doing it manually I did not see this anywhere, so I decided to remove it as eventually it would fail somewhere here if `Then I wait until I see "Target systems (1)" text` turns out to not be enough:
```
    When I click on run
    And I wait until I see "show response" text
    And I expand the results for "ssh_minion"
    Then I should see "My remote command output" in the command output for "ssh_minion"
```

Reply:
We should not have a timeout on `I wait until I do not see "pending" text` if the text is not around.

Reply:
This test has been tested in BV multiple time and should work : 
https://github.com/uyuni-project/uyuni/blob/09c6af7be4525f1d1edb8fc584b83d232c541378/testsuite/features/build_validation/smoke_tests/smoke_tests.template#L102-L113

Reply:
We can still use the `wait until` because the method I added for the click review should go. We could also wait for the button to switch  to `run` stage.

Reply:
Note that this step is also waiting for that button to be available during `DEFAULT_TIMEOUT` seconds.

```
When(/^I click on run$/) do
  find('button#run', wait: DEFAULT_TIMEOUT).click
end
```

Reply:
The code suggested by @maximenoel8 looks good to me:
```
   Then I should see a "Target systems (1)" text 
   When I wait until I do not see "pending" text 
   And I click on run 
```
I still don't get what prevents from using it.

Reply:
Indeed, by adding this line everything seems to be correct. I thought my failing attempts were from different reasons:
```
  Scenario: Run a remote command on this SSH tunnel minion                              # features/secondary/min_ssh_tunnel.feature:71
      This scenario ran at: 2024-08-13 14:26:13 +0200
    When I follow the left menu "Salt > Remote Commands"                                # features/step_definitions/navigation_steps.rb:339
    Then I should see a "Remote Commands" text in the content area                      # features/step_definitions/navigation_steps.rb:11
    When I enter command "echo 'My remote command output'"                              # features/step_definitions/salt_steps.rb:210
    And I enter the hostname of "ssh_minion" as "target"                                # features/step_definitions/navigation_steps.rb:437
      The hostname of ssh_minion is uyuni-master-podman-minssh-suse.mgr.suse.de
The run button is visible.
    And I click on preview                                                              # features/step_definitions/salt_steps.rb:171
    Then I should see a "Target systems (1)" text                                       # features/step_definitions/navigation_steps.rb:591
    When I wait until I do not see "pending" text                                       # features/step_definitions/navigation_steps.rb:43
    And I click on run                                                                  # features/step_definitions/salt_steps.rb:201
    And I wait until I see "show response" text                                         # features/step_definitions/navigation_steps.rb:39
    And I expand the results for "ssh_minion"                                           # features/step_definitions/salt_steps.rb:205
    Then I should see "My remote command output" in the command output for "ssh_minion" # features/step_definitions/salt_steps.rb:219
      This scenario took: 19 seconds
```
I changed it here with a new commit, and updated the port.

Reply:
cool